ql/src/java/org/apache/hadoop/hive/ql/parse/AlterTableSnapshotRefSpec.java

```diff
@@ -122,4 +122,29 @@ public String toString() {
       return MoreObjects.toStringHelper(this).add("refName", refName).add("ifExists", ifExists).toString();
     }
   }
+
+  public static class RenameBranchSpec {
```

Comment:
Also here, should we use `RenameSnapshotrefSpec`?

Reply:
Done